app/src/main/res/values/strings.xml

```diff
@@ -1133,10 +1133,7 @@
     The first parameter is the name of the app (e.g. Firefox Preview) -->
     <string name="onboarding_header">Welcome to %s!</string>
     <!-- text for the Firefox account onboarding sign in card header. The word "Firefox" should not be translated -->
-    <string name="onboarding_account_sign_in_header_1" moz:RemovedIn="99" tools:ignore="UnusedResources">Sync Firefox between devices</string>
-    <!-- text for the Firefox account onboarding sign in card header. The first parameter is the name of the application.-->
-    <string name="onboarding_account_sign_in_header_2">Sync %1$s between devices</string>
```

Comment:
We need to add `moz:RemovedIn="100" tools:ignore="UnusedResources"` since the strings made it into 99 https://github.com/mozilla-mobile/fenix/wiki/Removing-strings